darts/models/components/layer_norm_variants.py

```diff
@@ -54,3 +54,82 @@ def __init__(self, input_size, **kwargs):
 class LayerNorm(nn.LayerNorm):
     def __init__(self, input_size, **kwargs) -> None:
         super().__init__(input_size, **kwargs)
+
+
+class ReversibleInstanceNorm(nn.Module):
```

Comment:
I wonder if `RINorm` or `RevINorm` would be better names, as for longer ones such as RMSNorm we also use the abbreviation.
what do you think?

Reply:
I had been writing RIN, but RINorm is cleaner and feels more in line with RMSNorm.